### PR TITLE
Add strings for link-count

### PR DESCRIPTION
**Pull request by MatonAnthony**

### CODE REVIEW

File: add-on/_locales/en-US/messages.json [hunk 1]
```diff
@@ -354,5 +354,25 @@
   "autoExpandErrors_description": {
     "message": "Immediately display errors and warnings returned by a test when running the test suite.",
     "description": "Description of 'autoExpandErrors' property (shown in Add-ons Manager)"
+  },
+  "link_count": {
+    "message": "Link Count",
```
Comment (Elchi3) [inline add-on/_locales/en-US/messages.json]: Link count

File: add-on/_locales/en-US/messages.json [hunk 2]
```diff
@@ -354,5 +354,25 @@
...
+    "description": "Name of the rule"
+  },
+  "link_count_desc": {
+    "message": "Number of links in the document",
+    "description": "Number of links"
+  },
+  "count_link_info": {
+    "message": "There is %s links",
```
Comment (Elchi3) [inline add-on/_locales/en-US/messages.json]: There are

File: add-on/_locales/en-US/messages.json [hunk 3]
```diff
@@ -354,5 +354,25 @@
...
+    "description": "Informations when there is less than 100 links"
```
Comment (Elchi3) [inline add-on/_locales/en-US/messages.json]: Information when there are less than 100 links

File: add-on/_locales/en-US/messages.json [hunk 4]
```diff
@@ -354,5 +354,25 @@
...
+  },
+  "count_link_warning": {
+    "message": "There should be less than 100 links, currently : %s",
```
Comment (Elchi3) [inline add-on/_locales/en-US/messages.json]: no colon space :)

File: add-on/_locales/en-US/messages.json [hunk 5]
```diff
@@ -354,5 +354,25 @@
...
+    "description": "Warning regarding the number of links"
+  },
+  "count_link_error": {
+    "message": "There should be less than 250 links and ideally less than 100 links, currently : %s",
```
Comment (Elchi3) [inline add-on/_locales/en-US/messages.json]: no colon space :)

**Comment by MatonAnthony** [comment]
Yes, I'm sure it will happen once or twice at the beginning.

**Pull request merged**